src/pages/console/formbuilder/overrides.mdx

```diff
@@ -1,9 +1,10 @@
 export const meta = {
-  title: "Customize input element in code",
-  description: "Use the \"overrides\" property to override any form input's properties. Use this as an escape hatch in case there's a property that you can't customize within Studio."
-}
+  title: 'Customize input element in code',
+  description:
+    "Use the \"overrides\" property to override any form input's properties. Use this as an escape hatch in case there's a property that you can't customize within Studio."
+};
 
-Amplify Studio Forms are all based on top of the [Amplify UI library](https://ui.docs.amplify.aws/). The generated form provides a mechanism to override properties for each individual input component.
+[Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) Forms are all based on top of the [Amplify UI library](https://ui.docs.amplify.aws/). The generated form provides a mechanism to override properties for each individual input component.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio